sdk/eventhub/event-hubs/test/tsconfig.test.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs"
+  }
+}
```

Comment:
Why was this needed?

Reply:
Using tsconfig.test here `"integration-test:node"` in package.json